### PR TITLE
Renaming and refactor

### PR DESCRIPTION
**Pull request by ereslibre**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -16,61 +18,42 @@ pub extern "C" fn wapc_init() {
...
-
-    Ok(search_result.contains(&Rc::new(jmespatch::Variable::Bool(true))))
+fn has_privileged_container(containers: Option<Vec<apicore::Container>>) -> bool {
+    containers
+        .unwrap_or_default()
+        .into_iter()
+        .any(|container| {
+            container.security_context
+                .map_or(false, |security_context| {
+                    security_context.allow_privilege_escalation.unwrap_or(false)
```
Comment (ereslibre) [inline src/lib.rs]: I "copied" this logic as is. There's also the `privileged` attribute on the security context. Are we ignoring it in this policy in purpose? cc/ @flavio

Comment (flavio) [reply src/lib.rs]: That's intended, we have another policy looking after the creation of privileged containers

**Review by flavio — CHANGES_REQUESTED** [review]
Can you update the release action for github?

**Comment by ereslibre** [comment]
> Can you update the release action for github?

Done.

**Review by flavio — APPROVED** [review]
Great, please push a new tag with the new version we just changed inside of the Cargo.toml

**Pull request merged**